org.nanuc.Axolotl.yml

```diff
@@ -39,168 +29,73 @@ finish-args:
-          tag-pattern: "^v([\\d.]+)$"
-
-  - name: astilectron-bundler-astilectron
-    buildsystem: simple
-    build-options:
-      env:
-        BUNDLER_CACHE: /app/lib/astibundler/cache
-    build-commands:
-      - install -Dm 644 astilectron-0.56.0.zip ${BUNDLER_CACHE}/astilectron-0.56.0.zip
-    sources:
-      - type: file
-        url: https://github.com/asticode/astilectron/archive/v0.56.0.zip
-        sha256: 2725492843edaef0530543c041c5c7d9e43d4163f5c192fc8da11ce89fcaa0f5
-        dest-filename: astilectron-0.56.0.zip
-
-  - name: astilectron-bundler-electron-amd64
-    buildsystem: simple
-    only-arches:
-      - x86_64
-    build-options:
-      env:
-        BUNDLER_CACHE: /app/lib/astibundler/cache
-    build-commands:
-      - install -Dm 644 electron-linux-amd64-20.2.0.zip ${BUNDLER_CACHE}/electron-linux-amd64-20.2.0.zip
-    sources:
-      - type: file
-        url: https://github.com/electron/electron/releases/download/v20.2.0/electron-v20.2.0-linux-x64.zip
-        sha256: c0e3522de34819b838f4a35ddf30c6283c61be1bb8dff02089cda3f641938aad
-        dest-filename: electron-linux-amd64-20.2.0.zip
-
-  - name: astilectron-bundler-electron-arm64
-    buildsystem: simple
-    only-arches:
-      - aarch64
-    build-options:
-      env:
-        BUNDLER_CACHE: /app/lib/astibundler/cache
-    build-commands:
-      - install -Dm 644 electron-linux-arm64-20.2.0.zip ${BUNDLER_CACHE}/electron-linux-arm64-20.2.0.zip
-    sources:
-      - type: file
-        url: https://github.com/electron/electron/releases/download/v20.2.0/electron-v20.2.0-linux-arm64.zip
-        sha256: 1d2dd05f060786eae5ce687794aa071df8a5f5733b7b2efa1481de8db75d7738
-        dest-filename: electron-linux-arm64-20.2.0.zip
-
-  - name: axolotl-electron-bundle
-    buildsystem: simple
-    build-options:
-      env:
-        npm_config_offline: "true"
-        npm_config_cache: /run/build/axolotl-electron-bundle/flatpak-node/npm-cache
-        GOBIN: /app/bin
-        GO111MODULE: off
-        GOPATH: /run/build/axolotl-electron-bundle
-    subdir: src/github.com/nanu-c/axolotl
-    build-commands:
-      - make build-dependencies-axolotl-web
       - make build-axolotl-web
-      # Specify bundler working directory (where Electron and Astilectron is)
-      - echo "$(jq '. + { working_directory_path:"/app/lib/astibundler" }' bundler.json)" > bundler.json
-      - make build-axolotl-electron-bundle &
-        while [ -e /proc/$! ]; do echo -n "."; sleep 100; done
-      - make install-axolotl-electron-bundle DESTDIR=${FLATPAK_DEST} INSTALL_PREFIX=/bin
+      - make APP_ID=${FLATPAK_ID} DESTDIR=${FLATPAK_DEST} PREFIX="" install
     sources:
-      - generated-axolotl-web-sources.0.json
-      - generated-axolotl-sources.json
       - type: git
-        url: https://github.com/nanu-c/axolotl
-        tag: v1.6.0
-        commit: e521ab32b5bf315866a17f6844e5339abbc41e79
-        dest: src/github.com/nanu-c/axolotl
+        url: https://github.com/nanu-c/axolotl.git
```

Comment:
something for later: we dont need the `.git` for repository URLs.

Reply:
As different content is delivered for website and git+https, I like to differentiate these two optically as well even though I know that nowadays there is enough magic happening that the right application automatically receives the right content. I don't have a strong opinion on this though so I can remove it.

Reply:
aha! I have only seen manifests without the .git so far, but if it has a purpose then of course we should keep it. My argument for removing it was more asthetic - its less characters in the file and more compact.

Reply:
We keep it 👍